omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ModuleActionController1_8Tests.java

```diff
@@ -94,6 +97,21 @@ public void shouldDoNothingIfAtlasModuleAlreadyStarted() throws Exception {
 	}
 	
 	@Test
+	public void shouldUpdateModule() throws Exception {
+		String currentVersion = atlasModule.getVersion();
+		// To confirm the current version
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject module = deserialize(handle(req));
+		Assert.assertEquals(currentVersion, module.get("version"));
+
+		SimpleObject updatedModule = deserialize(handle(newPostRequest(getURI(), "{\"action\":\"update\", \"modules\":[\""
+		        + getUuid() + "\"], " +
```

Comment:
Would this test pass if one compiles the code while not connected to the internet?

Reply:
Nope, there will be an exception for the network failure. So the test case will fail